src/lib/layout/unauthenticated.svelte

```diff
@@ -127,4 +131,50 @@
         -webkit-background-clip: text;
         -webkit-text-fill-color: transparent;
     }
+
+    .aw-eyebrow {
+        font-family: 'Source Code Pro';
```

Comment:
We don't have `Aeonik Fono` in the Console. Is that okay?

Reply:
yes, we actually decided to use the operating systems font for `monospace`

Reply:
I added a fallback for monospace.